chapters/dashboard/hub.md

```diff
@@ -0,0 +1,14 @@
+# Jina Hub
+
+**[Jina Hub](https://dashboard.jina.ai/#/hub)** is a centralized registry, where you can share and discover custom and community driven Jina Pods and Apps tailored to specific use cases of neural search.
+
+# Features
+
+-  ##  Upload and Share Images
+    Share your custom Jina Pods by simply uploading a Python file, a Dockerfile and a YAML, and leave the rest of the work to us. Building and version control is automatically handled on the cloud side.
+
+-  ##  Cloud Native
+    As Jina Hub images are containerized, they can be easily leveraged in any Jina Flow, even in distributed environments.
+
+-  ##  Open Source
+    Hub images are made and maintained by the community.
```

Comment:
Add one more sentence, maybe 'You too can contribute to the database of hub images and help your fellow developers achieve their search goals!' to make it look less like we run out of ideas for what to write @lusloher any ideas for completing this?

Reply:
Added this and a link to Hub GitHub